.github/STYLEGUIDE.md

```diff
@@ -743,6 +743,37 @@ Each guide is broken into a few sections. Sections contain a list of guidelines.
   ```
   </details>
 
+* **PREFER attributes on the same line as what they are attributing.**
+
+  <details>
+
+  #### Why?
+
+    Because we naturally read lines of code as a consistent line, introducing whitespace makes it that much harder to understand the full context of the line you're trying to understand. This is the same reasoning we put behind preferring a return on the same line as the final parameter in a function declaration.
+
+    NOTE: This does not come at the cost of sanity. If you have an exceptionally large number of attributions then it might indicate that you've gotten overzealous, and it actually obscures your meaning, rather than add to it.
```

Comment:
Should we call out the `@available` attribute?

Reply:
Calling out splitting into 2 potential guidelines one for inline when attributes are on functions, properties, or closures and another for attribute on the line above for classes, structs, and actors.

`@available` is still an outlier in that it always goes on its own line, so yes `@available` makes sense to be called out specifically at least in regards to inline preference.

Reply:
I think 2 rules make sense.  We should also show an example of what they look like with multiple attributes so that I know if I'm inlining on a separate line or stacking them.  And I think `@available` is an exception entirely and is always on its own line by itself.